dev-support/docker/Dockerfile

```diff
@@ -110,6 +110,13 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN curl --location --fail --silent --show-error --output /tmp/adoptopenjdk11.tar.gz "${OPENJDK11_URL}" && \
   echo "${OPENJDK11_SHA256} */tmp/adoptopenjdk11.tar.gz" | sha256sum -c -
 
+FROM base_image AS openjdk17_download_image
+ENV OPENJDK17_URL 'https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.9%2B9/OpenJDK17U-jdk_x64_linux_hotspot_17.0.9_9.tar.gz'
```

Comment:
https://github.com/adoptium/temurin17-binaries/releases/tag/jdk-17.0.10%2B7

I think 17.0.10 is the latest release? Not sure why 17.0.9 is on the top...

Reply:
Since 17.0.9 on top picked it. Thanks for pointing correcting one @Apache9 . Changed it.